src/test/java/com/wikia/webdriver/testcases/desktop/spamwikireview/ViewSpamWikisTests.java

```diff
@@ -0,0 +1,150 @@
+                           + SpamWikiReviewPage.LANGUAGE_CODE.all);
+  }
+
+
+  @Test
+  @Execute(asUser = User.STAFF)
+  public void filterOtherLanguagesTest() {
+    SpamWikiReviewPage spamWikiReviewPage = new SpamWikiReviewPage();
+    spamWikiReviewPage.open();
+
+    String spamWikiReviewListViewUrl = spamWikiReviewPage.getCurrentUrl();
+
+    // select 'ja' language
+    spamWikiReviewPage.selectLanguageOfWikis(SpamWikiReviewPage.LANGUAGE_CODE.other);
+
+
+    // assert current page url contains query param selecting the language
+    Assertion.assertEquals(spamWikiReviewPage.getCurrentUrl(),
+                           spamWikiReviewListViewUrl + SpamWikiReviewPage.QUERY_PARAM_LANG
+                           + SpamWikiReviewPage.LANGUAGE_CODE.other);
+
+    // check if all Wikis on a presumably filtered page have languages other than those in LANGUAGE_CODE
+    for(WebElement wikiRow : spamWikiReviewPage.getListDisplayedWikisTableRows()) {
+      Assertion.assertFalse(EnumUtils.isValidEnum(SpamWikiReviewPage.LANGUAGE_CODE.class,
+                                                 wikiRow.findElement(By.xpath(SpamWikiReviewPage.XPATH_LANGUAGE_COLUMN))
+                                                     .getText()),
+                           "Spam Wiki Review other languages view contained languages from"
+                           + "LANGUAGE CODE - those that can be filtered individually, "
+                           + "namely " + wikiRow.findElement(By.xpath(SpamWikiReviewPage.XPATH_LANGUAGE_COLUMN))
+                               .getText());
+    }
+  }
+
+  @Test
+  @Execute(asUser = User.STAFF)
+  public void ShowOnlyQuestionableWikisByNavigationTest() {
+    SpamWikiReviewPage spamWikiReviewPage = new SpamWikiReviewPage();
+    spamWikiReviewPage.open();
+
+    // navigate to Questionable Wikis page
+    spamWikiReviewPage.showQuestionableWikis();
+
+    // assert current page url contains query param selecting the questionable Wikis
+    Assertion.assertStringContains(spamWikiReviewPage.getCurrentUrl(), "status=questionable");
+
+    // check if all Wikis on a presumably only questionable wikis page
+    for(WebElement wikiRow : spamWikiReviewPage.getListDisplayedWikisTableRows()) {
+      String visibleButtonsText = wikiRow.findElement(By.xpath(SpamWikiReviewPage.XPATH_STATUS_COLUMN)).getText();
+
+      //have Spam and Not Spam buttons in that order without Questionable button
+
+      Assertion.assertStringNotContains(visibleButtonsText,
+                                        SpamWikiReviewPage.QUESTIONABLE_BUTTON_TEXT);
+
+      Assertion.assertEquals(visibleButtonsText,SpamWikiReviewPage.SPAM_BUTTON_TEXT
+                                                + "\n" + SpamWikiReviewPage.NOT_SPAM_BUTTON_TEXT);
+    }
+
+
+
+  }
+
+  @Test
+  @Execute(asUser = User.STAFF)
+  public void SpamWikiReviewHeaderNavigatesToMainPageTest() {
+    // TODO: Click on Spam Wiki Review in header and see if it redirects to main page
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)